src/main/webapp/static/js/service/gameService.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+angular.module('myApp').factory('GameService', GameServiceFactory)
+
+GameServiceFactory.$inject = ['$http', '$log']
+
+function GameServiceFactory($http, $log) {
+	var REST_SERVICE_URI = 'http://localhost:8080/GameRate/game/';
+
+    var factory = {
+        fetchAllGames: fetchAllGames,
+        createGame: createGame,
+        updateGame:updateGame,
+        deleteGame:deleteGame
+    };
+
+    return factory;
+
+    function fetchAllGames() {
+        return $http.get(REST_SERVICE_URI).then(
+			function (response) {
+                return response.data;
+            },
+            function (errResponse) {
+                $log.error('Error while fetching Games ', errResponse);
+            }
+        );
+    }
+
+    function createGame(game) {
+        return $http.post(REST_SERVICE_URI, game).then(
+            function (response) {
+                return response.data;
+            },
+            function(errResponse){
+                $log.error('Error while creating Game', errResponse);
+            }
+        );
+    }
+
+
+    function updateGame(game, id) {
+        return $http.put(REST_SERVICE_URI + id, game).then(
```

Comment:
So going back to my comment earlier about the /{id} thing. You can setup this call to be more safe by adding the id to the game object and running validation in your java service for it